src/rgw/librgw.cc

```diff
@@ -78,6 +78,12 @@ static void handle_sigterm(int signum)
   dout(20) << __func__ << " SIGUSR1 ignored" << dendl;
 }
 
+#ifdef HAVE_JAEGER
+ thread_local tracing::Tracer tracer(jaeger_configuration::jaeger_rgw_config);
+#else
+ tracing::Tracer tracer;
+#endif
```

Comment:
i don't think this global variable should be duplicated in `rgw_main.cc` and `librgw.cc`. can we just keep it in `rgw/rgw_tracer.cc`?

Reply:
we removed `rgw/rgwtracer.cc`
do you want to add it back for the global var?

Reply:
yes, i'd like to keep rgw_tracer.* for the rgw-specific trace strings, jaeger config, and global variable